### PR TITLE
chat: update room listing

### PR DESCRIPTION
**Pull request by Mikaela**

### CODE REVIEW

File: chat/index.html [hunk 1]
```diff
@@ -70,24 +70,29 @@ <h4>What happens when I remove a message in Riot?</h4>
...
 
-<h2 class="pt-4">Channels</h2>
-<p>To join a channel, just type <code>/join [channel name]</code> in any room. You can run this command from any server, not just ours!</p>
-<p>The privacytools.io administration operates the following channels:</p>
+<h2 class="pt-4">Rooms</h2>
+<p>To join a room, just type <code>/join [room address]</code> in any room. You can run this command from any server, not just ours!</p>
+<p>The PrivacyTools administration operates the following channels:</p>
 <ul>
+  <li><code>#forum:privacytools.io</code>: All forum.privacytoools.io updates.</li>
   <li><strong><code>#general:privacytools.io</code>: Main discussion channel.</strong> All members of our homeserver will auto-join here.</li>
```
Comment (Mikaela) [inline chat/index.html]: I wanted to keep alphabedical order

File: chat/index.html [hunk 2]
```diff
@@ -70,24 +70,29 @@ <h4>What happens when I remove a message in Riot?</h4>
...
   <li><code>#github:privacytools.io</code>: GitHub updates for @privacytoolsIO.</li>
   <li><code>#guides:privacytools.io</code>: Privacy Guides development work.</li>
   <li><code>#team:privacytools.io</code>: Private.</li>
```
Comment (Mikaela) [inline chat/index.html]: Could be explained better

File: chat/index.html [hunk 3]
```diff
@@ -70,24 +70,29 @@ <h4>What happens when I remove a message in Riot?</h4>
...
+  <li><code>#wiki:privacytools.io</code>: wiki.privacytools.io discussion room</li>
 </ul>
 <p>If you're looking for more things to chat about, check out these channels too:</p>
 <ul>
-  <li><code>#matrix:matrix.org</code>: Discuss Matrix itself!</li>
-  <li><code>#disroot:disroot.org</code></li>
+  <li><code>#anime:matrix.org</code> discussion about anime and manga</li>
+  <li><code>#disroot:disroot.org</code>: Disroot.org room</li>
   <li><code>#freenode_#opennic:matrix.org</code>: OpenNIC IRC Channel.</li>
-  <li><code>#TorProject:privacytools.io</code>: Unofficial Tor Project chat room!</li>
```
Comment (Mikaela) [inline chat/index.html:87]: Why unofficial when official also exists?

File: chat/index.html [hunk 4]
```diff
@@ -70,24 +70,29 @@ <h4>What happens when I remove a message in Riot?</h4>
...
 <p><a href="https://github.com/matrix-org/matrix-bifrost/issues/63">It's an issue with the XMPP bridge</a>.</p>
 
-<h2 class="pt-4">Channels</h2>
-<p>To join a channel, just type <code>/join [channel name]</code> in any room. You can run this command from any server, not just ours!</p>
-<p>The privacytools.io administration operates the following channels:</p>
+<h2 class="pt-4">Rooms</h2>
+<p>To join a room, just type <code>/join [room address]</code> in any room. You can run this command from any server, not just ours!</p>
+<p>The PrivacyTools administration operates the following channels:</p>
 <ul>
+  <li><code>#forum:privacytools.io</code>: All forum.privacytoools.io updates.</li>
```
Comment (jonaharagon) [inline chat/index.html:78]:
```suggestion
  <li><code>#dev:privacytools.io</code>: Discussions relating to building www.privacytools.io.</li>
  <li><code>#forum:privacytools.io</code>: All forum.privacytoools.io updates.</li>
```

File: chat/index.html [hunk 5]
```diff
@@ -70,24 +70,30 @@ <h4>What happens when I remove a message in Riot?</h4>
...
+  <li><code>#matrix:matrix.org</code>: Discuss Matrix itself!</li>
   <li><code>#nothing_to_hide:privacytools.io</code>: More general privacy chat.</li>
+  <li><code>#_oftc_#Tor-Project:matrix.org</code>: Official Tor Project chat room!</li>
   <li><code>#plume:disroot.org</code>: Discussion for Plume: Federated blogging.</li>
+  <li><code>#riot-android:matrix.org</code>: Riot-Android discussion and support.</li>
+  <li><code>#riot-ios:matrix.org</code>: Riot-iOS discussion and support.</li>
+  <li><code>#riotx:matrix.org</code>: RiotX discussion and support.</li>
   <li><code>#riot-web:matrix.org</code>: Riot-web discussion and support.</li>
 </ul>
 <p>If you are a room operator and want your channel listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
<p>If you are a room operator and want your room listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
```

File: chat/index.html [hunk 6]
```diff
@@ -70,24 +70,30 @@ <h4>What happens when I remove a message in Riot?</h4>
...
 <ul>
+  <li><code>#dev:privacytools.io</code>: Discussions relating to building www.privacytools.io.</li>
+  <li><code>#forum:privacytools.io</code>: All forum.privacytoools.io updates.</li>
   <li><strong><code>#general:privacytools.io</code>: Main discussion channel.</strong> All members of our homeserver will auto-join here.</li>
   <li><code>#github:privacytools.io</code>: GitHub updates for @privacytoolsIO.</li>
   <li><code>#guides:privacytools.io</code>: Privacy Guides development work.</li>
   <li><code>#team:privacytools.io</code>: Private.</li>
+  <li><code>#wiki:privacytools.io</code>: wiki.privacytools.io discussion room</li>
 </ul>
 <p>If you're looking for more things to chat about, check out these channels too:</p>
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
<p>If you're looking for more things to chat about, check out these rooms as well:</p>
```

File: chat/index.html [hunk 7]
```diff
@@ -70,24 +70,30 @@ <h4>What happens when I remove a message in Riot?</h4>
...
-<h2 class="pt-4">Channels</h2>
-<p>To join a channel, just type <code>/join [channel name]</code> in any room. You can run this command from any server, not just ours!</p>
-<p>The privacytools.io administration operates the following channels:</p>
+<h2 class="pt-4">Rooms</h2>
+<p>To join a room, just type <code>/join [room address]</code> in any room. You can run this command from any server, not just ours!</p>
+<p>The PrivacyTools administration operates the following channels:</p>
 <ul>
+  <li><code>#dev:privacytools.io</code>: Discussions relating to building www.privacytools.io.</li>
+  <li><code>#forum:privacytools.io</code>: All forum.privacytoools.io updates.</li>
   <li><strong><code>#general:privacytools.io</code>: Main discussion channel.</strong> All members of our homeserver will auto-join here.</li>
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
  <li><strong><code>#general:privacytools.io</code>: Main discussion room.</strong> All members of our homeserver will auto-join here.</li>
```

File: chat/index.html [hunk 8]
```diff
@@ -70,27 +70,33 @@ <h4>What happens when I remove a message in Riot?</h4>
...
+  <li><code>#_oftc_#Tor-Project:matrix.org</code>: Official Tor Project chat room!</li>
   <li><code>#plume:disroot.org</code>: Discussion for Plume: Federated blogging.</li>
+  <li><code>#riot-android:matrix.org</code>: Riot-Android discussion and support.</li>
+  <li><code>#riot-ios:matrix.org</code>: Riot-iOS discussion and support.</li>
+  <li><code>#riotx:matrix.org</code>: RiotX discussion and support.</li>
   <li><code>#riot-web:matrix.org</code>: Riot-web discussion and support.</li>
 </ul>
-<p>If you are a room operator and want your channel listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
+<p>If you are a room operator and want your room listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
 <p>We do not necessarily endorse any of the content in these listed channels.</p>
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
<p>We do not necessarily endorse any of the content in these listed rooms.</p>
```

File: chat/index.html [hunk 9]
```diff
@@ -70,28 +70,34 @@ <h4>What happens when I remove a message in Riot?</h4>
...
+  <li><code>#riotx:matrix.org</code>: RiotX discussion and support.</li>
   <li><code>#riot-web:matrix.org</code>: Riot-web discussion and support.</li>
 </ul>
-<p>If you are a room operator and want your channel listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
-<p>We do not necessarily endorse any of the content in these listed channels.</p>
+<p>If you are a room operator and want your room listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
+<p>We do not necessarily endorse any of the content in these listed rooms.</p>
 
 <h2 class="pt-4" id="contact">Contact</h2>
 <p>For server related issues, the administrator of this homeserver is Jonah, who can be reached on Matrix at <code>@jonah:privacytools.io</code> or via email at <code>jonah@privacytools.io</code>.
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
<p>For server related issues, the administrator of this homeserver is Jonah, who can be reached on Matrix at <code>@jonah:privacytools.io</code>.
```

File: chat/index.html [hunk 10]
```diff
@@ -70,30 +70,36 @@ <h4>What happens when I remove a message in Riot?</h4>
...
 </ul>
-<p>If you are a room operator and want your channel listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
-<p>We do not necessarily endorse any of the content in these listed channels.</p>
+<p>If you are a room operator and want your room listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
+<p>We do not necessarily endorse any of the content in these listed rooms.</p>
 
 <h2 class="pt-4" id="contact">Contact</h2>
-<p>For server related issues, the administrator of this homeserver is Jonah, who can be reached on Matrix at <code>@jonah:privacytools.io</code> or via email at <code>jonah@privacytools.io</code>.
+<p>For server related issues, the administrator of this homeserver is Jonah, who can be reached on Matrix at <code>@jonah:privacytools.io</code>.
 <p>For moderation related issues, you can contact Jonah or anyone in the <code>+team:privacytools.io</code> community on Matrix.</p>
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
<p>For moderation related issues, you can contact Jonah or anyone in the team (listed in the <code>+team:privacytools.io</code> community) via Matrix, or via email at <code>support@privacytools.io</code>.</p>
```

File: chat/index.html [hunk 11]
```diff
@@ -70,30 +70,36 @@ <h4>What happens when I remove a message in Riot?</h4>
...
-<p>We do not necessarily endorse any of the content in these listed channels.</p>
+<p>If you are a room operator and want your room listed here, feel free to contact us. Especially if you operate a large room and/or are a member of our homeserver!</p>
+<p>We do not necessarily endorse any of the content in these listed rooms.</p>
 
 <h2 class="pt-4" id="contact">Contact</h2>
-<p>For server related issues, the administrator of this homeserver is Jonah, who can be reached on Matrix at <code>@jonah:privacytools.io</code> or via email at <code>jonah@privacytools.io</code>.
-<p>For moderation related issues, you can contact Jonah or anyone in the <code>+team:privacytools.io</code> community on Matrix.</p>
+<p>For server related issues, the administrator of this homeserver is Jonah, who can be reached on Matrix at <code>@jonah:privacytools.io</code>.
+<p>For moderation related issues, you can contact Jonah or anyone in the team (listed in the <code>+team:privacytools.io</code> community) via Matrix, or via email at <code>support@privacytools.io</code>.</p>
 <p>For anything else, visit <a href="https://www.privacytools.io/contact/">www.privacytools.io/contact</a>.</p>
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
<p>For moderation related issues or anything else, you can contact Jonah or anyone in the team (listed in the <code>+team:privacytools.io</code> community) via Matrix, or via email at <code>support@privacytools.io</code>.</p>
```

File: chat/index.html [hunk 12]
```diff
@@ -70,30 +70,35 @@ <h4>What happens when I remove a message in Riot?</h4>
...
+<p>To join a room, just type <code>/join [room address]</code> in any room. You can run this command from any server, not just ours!</p>
+<p>The PrivacyTools administration operates the following channels:</p>
 <ul>
-  <li><strong><code>#general:privacytools.io</code>: Main discussion channel.</strong> All members of our homeserver will auto-join here.</li>
+  <li><code>#dev:privacytools.io</code>: Discussions relating to building www.privacytools.io.</li>
+  <li><code>#forum:privacytools.io</code>: All forum.privacytoools.io updates.</li>
+  <li><strong><code>#general:privacytools.io</code>: Main discussion room.</strong> All members of our homeserver will auto-join here.</li>
   <li><code>#github:privacytools.io</code>: GitHub updates for @privacytoolsIO.</li>
   <li><code>#guides:privacytools.io</code>: Privacy Guides development work.</li>
   <li><code>#team:privacytools.io</code>: Private.</li>
```
Comment (jonaharagon) [inline chat/index.html]:
```suggestion
  <li><code>#team:privacytools.io</code>: Private team chat.</li>
  <li><code>#wiki:privacytools.io</code>: <a href="https://wiki.privacytools.io/view/Main_Page">Wiki</a>-building discussion room.</li>
```

File: chat/index.html [hunk 13]
```diff
@@ -70,30 +70,36 @@ <h4>What happens when I remove a message in Riot?</h4>
...
-  <li><strong><code>#general:privacytools.io</code>: Main discussion channel.</strong> All members of our homeserver will auto-join here.</li>
+  <li><code>#dev:privacytools.io</code>: Discussions relating to building www.privacytools.io.</li>
+  <li><code>#forum:privacytools.io</code>: All forum.privacytoools.io updates.</li>
+  <li><strong><code>#general:privacytools.io</code>: Main discussion room.</strong> All members of our homeserver will auto-join here.</li>
   <li><code>#github:privacytools.io</code>: GitHub updates for @privacytoolsIO.</li>
   <li><code>#guides:privacytools.io</code>: Privacy Guides development work.</li>
-  <li><code>#team:privacytools.io</code>: Private.</li>
+  <li><code>#team:privacytools.io</code>: Private team chat.</li>
+  <li><code>#wiki:privacytools.io</code>: <a href="https://wiki.privacytools.io/view/Main_Page">Wiki</a>-building discussion room.</li>
+  <li><code>#wiki:privacytools.io</code>: wiki.privacytools.io discussion room</li>
```
Comment (jonaharagon) [inline chat/index.html:84]: Oh, I am blind :)

**Pull request merged**